docs/man-openmpi/man3/ERRORS.rst

```diff
@@ -0,0 +1,49 @@
+Almost all MPI routines return an error value; C routines as the return result
```

Comment:
Is there a reason not to put the "ERRORS" section title in this file, too?

Reply:
I didn't think of putting the ERRORS header in this file. 

But, as I was going thru the individual manpage files, I noticed some pages had headings included ERRORS in all caps and some as Errors.

If I put the ERRORS heading in this file, then I think it's going to look odd when all the other headings are mixed case on a page.

I can change it if you want it changed anyway.

Reply:
The man pages were written over a long period of time, by different people.  It's possible (likely!) that there are inconsistencies like this (ERRORS vs. Errors).

Let's make new text as consistent as possible (and potentially fix old inconsistencies).  I.e., make the Errors subsection follow whatever the convention is for that level of subsection.  Are there other titles of the same level of subsection that are inconsistent?

Reply:
I'm not sure what you're asking to do about the inconsistency regarding ERRORS vs Errors. If I move that into the ERRORS.rst file, then I guess I would use ERRORS there since that is a section heading. But then do I find the other section headings in every man page including the ones I have not yet touched and change all of those to uppercase?

I agree overall formatting consistency is desirable, but I also think consistency within a page is more important than overall consistency.

Should I be changing all section headings in this pull request? That feels a bit out of scope for the intent of this pull request.

Reply:
Yes, I'm suggesting moving the subsection title into ERRORS.rst.  I'm further suggesting make the capitalization of the subsection title in ERRORS.rst be consistent with whatever we do for other subsection titles at the same level in other man pages.

That being said, if we have more than just "ERRORS" vs. "Errors" inconsistencies in capitalization of subsection titles, we should probably make them *all* consistent.  That would satisfy both overall formatting consistency *and* within-one-man-page consistence.  I agree that this should probably be either a separate PR, or a standalone commit on this PR.  It probably wouldn't be too hard to hack up a script or two to help automate making the pages' subsection titles be consistent (vs. editing all bazillion of our man pages manually).  It might be worth looking at other popular man pages to see what their conventions are -- e.g., some GNU man pages like binutils or coreutils or something.

Reply:
@jsquyres I prefer to handle all the man page section headings changes, which I think can be scripted, as a separate pull request in order to keep that change from being tangled in more changes directly related to this pull request reviews. I also prefer to keep the ERRORS heading in each man page since there are some man pages with text about error handling which is different than what's described in ERRORS.rst. So I think keeping the ERRORS heading in each man page is more consistent and avoids getting anyone confused about where that heading went when ERRORS.rst is included.

Reply:
Per discussion - this will happen in a separate PR after this PR is merged